lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -638,6 +639,28 @@ def test_course_image(self):
         self.assertTrue(url.startswith('/static/toy_course_dir/'))
         self.course.static_asset_path = ""
 
+    def test_course_image_for_split_course(self):
+        """
+        if course_image is empty the course_image_url will be blank
+        """
+        migrator = SplitMigrator(
```

Comment:
See https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/tests/test_course_settings.py#L421-422 for how to create a split course using `CourseFactory`.

Also, like that test above, use ddt to confirm that this will not throw an error for a course created in split or in old mongo